README.asciidoc

```diff
@@ -54,6 +54,41 @@ Talk to other users of JBake on the forum:
 
 * http://groups.google.com/group/jbake-user[Forum]
 
+== Docker Image
+
+The image uses the official openjdk:8-alpine image for building a distribution of JBake and openjdk:8-jre-alpine for runtime.
```

Comment:
This doesn't look correct anymore. New change is using JDK11 to build and JRE11 to run.

Reply:
Ah yeah thanks for catching that.